CHANGELOG.md

```diff
@@ -7,6 +7,26 @@
   If needed, `StableConnectIdentites` can be disabled in the feature gates configuration in the Cluster Operator.
 * Support for the ppc64le platform
 
+### Changes, deprecations and removals
+* There are three ways to configure Cruise Control CPU capacity values:
+  * brokerCapacity (for all brokers)
+  * brokerCapacity overrides (per broker)
+  * Kafka resource requests and limits (for all brokers).
+* The precedence of which Cruise Control CPU capacity configuration is used has been changed.
+* In previous Strimzi versions, the Kafka resource limit (if set) took precedence, regardless if any other CPU configurations were set. 
+  * For example:
+    * (1) Kafka resource limits
+    * (2) brokerCapacity overrides
+    * (3) brokerCapacity
+* This previous behavior was identified as a bug and was fixed in this Strimzi release.
+* Going forward, the brokerCapacity overrides per broker take top precedence, then general brokerCapacity configuration, and then the Kafka resource requests, then the Kafka resource limits. 
+  * For example:
+     * (1) brokerCapacity overrides
+     * (2) brokerCapacity
+     * (3) Kafka resource requests
+     * (4) Kafka resource limits
+  * When none of Cruise Control CPU capacity configurations mentioned above are configured, CPU capacity will be set to `1`.
```

Comment:
You need to give it some structure as all of this is just one item on the list of changes. I think it might be also worth explaining what does `brokerCapacity` mean - maybe using `.spec.cruiseControl.brokerCapacity` if that is the path.

```suggestion
### Changes, deprecations and removals

* The automatic configuration of CRuise Control CPU capacity has been changed in this release:
  * There are three ways to configure Cruise Control CPU capacity values:
    * `.spec.cruiseControl.brokerCapacity` (for all brokers)
    * `.spec.cruiseControl.brokerCapacity` overrides (per broker)
    * Kafka resource requests and limits (for all brokers).
  * The precedence of which Cruise Control CPU capacity configuration is used has been changed.
  * In previous Strimzi versions, the Kafka resource limit (if set) took precedence, regardless if any other CPU configurations were set. 
    * For example:
      * (1) Kafka resource limits
      * (2) `.spec.cruiseControl.brokerCapacity` overrides
      * (3) `.spec.cruiseControl.brokerCapacity`
  * This previous behavior was identified as a bug and was fixed in this Strimzi release.
  * Going forward, the brokerCapacity overrides per broker take top precedence, then general brokerCapacity configuration, and then the Kafka resource requests, then the Kafka resource limits. 
    * For example:
       * (1) `.spec.cruiseControl.brokerCapacity` overrides
       * (2) `.spec.cruiseControl.brokerCapacity`
       * (3) Kafka resource requests
       * (4) Kafka resource limits
    * When none of Cruise Control CPU capacity configurations mentioned above are configured, CPU capacity will be set to `1`.
```

Reply:
@kyguy Is this better now? Thanks